src/components/Agentprofile/AgentProperties/AgentProperties.jsx

```diff
@@ -0,0 +1,97 @@
+import React, { useEffect, useState } from 'react'
+import axios from 'axios';
+import { Dashboard } from '../../Dashboard/Dashboard';
+import './style.css'
+import { useParams } from 'react-router-dom';
+import { useNavigate } from 'react-router-dom';
+
+const url = "http://localhost:4000/v1/agent/property";
+
+const AgentProperties = () => {
+
+ const navigate = useNavigate()
+
+    const token =  localStorage.getItem("token")
+
+    const config = {
+        headers : {
+            Authorization: token
+        }
+    }
+    const [agentProperties, setAgentProperties] = useState([])
+
+    const getAgentProperties = async () => {
+        const response = await axios.get(url, config)
+        const {data} = response
+        setAgentProperties(data)
+    }
+
+    useEffect(()=> {
+        getAgentProperties();
+    }, [])
+
+    const deleteItem = async(id) => {
+      const removeProperty = agentProperties.filter((item) => item.id !== id)
+setAgentProperties(removeProperty)
+      try {
+        const response = await axios.delete(`http://localhost:4000/v1/agent/property/${id}`, config)
```

Comment:
You shouldn't hardcode your API URL. It should be dynamic based on the environment you are on. When you deploy this code online, your API URL will no longer be localhost:3006, how will you solve the issue in that case?